tests/lookup/tests.py

```diff
@@ -776,6 +780,30 @@ def test_chain_date_time_lookups(self):
             ordered=False
         )
 
+    def test_comparison_against_decimal_field(self):
+        """
+        A comparison against an expression with DecimalField output_field
+        is handled correctly.  (#27498).
+        """
+        p1 = Product.objects.create(name="Product1", qty_target=10)
```

Comment:
Might make sense to put in `setupTestData` of a test test case -- if each lookup class has its own test method.
Use single quotes unless a string has a single quote in it.